common/lib/dependabot/pull_request_creator/labeler.rb

```diff
@@ -351,23 +351,24 @@ def create_gitlab_security_label
       end
 
       def create_github_language_label
-        langauge_name =
-          self.class.label_details_for_package_manager(package_manager).
-          fetch(:name)
+        label = self.class.label_details_for_package_manager(package_manager)
+        language_name = label.fetch(:name)
         github_client_for_source.add_label(
           source.repo,
-          langauge_name,
-          self.class.label_details_for_package_manager(package_manager).
-            fetch(:colour),
-          description: "Pull requests that update #{langauge_name.capitalize} "\
-                       "code",
+          language_name,
+          label.fetch(:colour),
+          description: label.fetch(:description) { default_description_for(language_name) },
```

Comment:
I chose to introduce an optional key `:description` key in the Hash to allow overriding the default description that is generated.